vpp/vaapipostprocess_scaler.cpp

```diff
@@ -369,6 +398,15 @@ VaapiPostProcessScaler::setParameters(VppParamType type, void* vppParam)
 
         return setColorBalanceParam(*colorbalance);
     }
+    else if (type == VppParamTypeTransform) {
```

Comment:
Could you add get function too?

Reply:
sorry about that @xuguangxin , I omitted that. 
It should be added with set function, but the interface "IVideoPostProcess" doesn't define it, and the other VPP attributes have not implemented getParameter() function, so how about to add this function in the next pull-request with other VPP attributes?

Reply:
then please remove get function....sorry for this